### ISSUE TITLE
Issue with "tel:" Links in GrapeJS Builder

### ISSUE THREAD

**Issue by ricfreire** [mautic/mautic#14197]
### Mautic Series

5.1.x series

### Mautic installed version

5.1.0

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.1

### What browsers are you seeing the problem on?

Chrome

### What happened?

When adding a telephone link `tel:` in a text block within the GrapesJS builder, the link value resets to `#` upon reopening the block. After closing and reopening the builder, the href attribute still displays as `#`, but a new option appears in the right-side panel to set the link again.

This issue may cause unexpected behavior and introduces unnecessary extra steps to ensure phone links are saved correctly.

Tested with `mailto:` and it works correctly.

### How can we reproduce this issue?

Step 1: Go to Channels > Emails
Step 2: Create a new email.
Step 3: Select any template (HTML or MJML, it doesn't matter).
Step 4: Open the builder.
Step 5: In a text block, open it and add the following link: `tel:+351000000` (feel free to use a real phone number if needed).
![image](https://github.com/user-attachments/assets/2cd3e788-61f4-4152-83d1-75a6c9af35c8)
Step 6: When done, hover the cursor over the link. It will correctly display the link you inserted. Save the text block.
Step 7: Open the same text block again. The link now appears as `#` instead of the original value. [BUG]
![image](https://github.com/user-attachments/assets/8cee04e7-caa7-4ddb-b2a5-00327f1dec64)
Step 8: Close the builder and reopen it.
Step 9: Hover over the link. Now it becomes clickable and editable through the right-side options. [BUG?]
![image](https://github.com/user-attachments/assets/93b9ea2e-0fdd-48f8-bed3-6883f854496b)
Step 10: Using the right-side options, change the link back to `tel:+351000000`. Save and close the builder.
Step 11: Preview the email. The phone number link now displays correctly after modifying the href through the right-side options.

### Relevant log output

```shell
None.
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by dsp76** [comment]
Confirmed... we experienced the same. Its probably related to the embedded CK Editor. I will find out how to handle this type of issue.

**Comment by seniorensachsen** [comment]
Alright - its an issue with the way Mautic uses the CKeditor. Will be fixed with Version 5.2 
https://github.com/mautic/mautic/pull/14154

**Comment by dsp76** [comment]
Confirmed! Will be fixed with v5.2

https://github.com/mautic/mautic/pull/14154

**Comment by LordRembo** [comment]
@ricfreire @dsp76 Can you confirm this issue is fixed? Looks like the linked changes were merged, and it should have been in the latest release (5.2.x).

**Comment by LordRembo** [comment]
@ricfreire @dsp76 I don't think it was, the linked item was closed before any protocol change suggestions were made.

**Comment by LordRembo** [comment]
Added a pull request to add those protocols: #14453

**Comment by dsp76** [comment]
At least since 5.2.1 its fixed. Just tested.
